db/migrate/20160719100630_create_wishlists.rb

```diff
@@ -0,0 +1,9 @@
+class CreateWishlists < ActiveRecord::Migration[5.0]
+  def change
+    create_table :wishlists do |t|
+      t.string :name
```

Comment:
If you want to enforce those validation even better, you can set the field na migration as required not to be false. Again, I think `:description` can be null, and `:name` mustn't.